scripts/startLocalCluster.sh

```diff
@@ -311,5 +311,5 @@ done
 
 echo == Done, your cluster is ready at
 for p in `seq $CO_BASE $PORTTOPCO` ; do
-    echo "   ${BUILD}/bin/arangosh --server.endpoint $TRANSPORT://[::1]:$p"
+    echo "   ${BUILD}/bin/arangosh --server.endpoint '$TRANSPORT://[::1]:$p'"
```

Comment:
This change was not present in the 3.7 and devel versions of the PR.
Shall it be added there too?

Reply:
doesn't really matter

Reply:
Ok